### DISCUSSION TITLE
what does tm_src and tm_tgt mean in PMPCorefinementVisitor::after_vertex_copy() function ?

### DISCUSSION THREAD

**Discussion by citystrawman** [CGAL/cgal#8298] [Q&A] [answered]
_Please use the following template to help us solving your issue._

## Issue Details

I am trying to use 
```
PMPCorefinementVisitor::after_vertex_copy(vertex_descriptor v_src, const Mesh& tm_src,
        vertex_descriptor  v_tgt , Mesh& tm_tgt)
```
in split method:
`PMP::split(tm, splitter, params::visitor(v), params::do_not_modify(true));`,
 however, I am not quite clear about the meaning of `tm_src `and `tm_tgt `in this function:

![image](https://github.com/CGAL/cgal/assets/9004748/bc6ae9ba-2d3e-44e9-8959-17ec791401e5)

As you can see, here  `tm_src `and `tm_tgt ` are both exactly the same in terms of their vertex as well as halfedges, and they both seems to be the same as `tm `in split function.   

Then my question is, since they are all same, why they are differentiated as  `tm_src `and `tm_tgt ` ?

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits):
* Compiler:
* Release or debug mode:
* Specific flags used (if any):
* CGAL version:
* Boost version:
* Other libraries versions if used (Eigen, TBB, etc.):

**Comment by sloriot** [comment, marked as answer]
For split, they are indeed the same. The visitor concept is used by several functions, it is not specific to split.

**Reply by citystrawman** [reply]
> For split, they are indeed the same. The visitor concept is used by several functions, it is not specific to split.

Thanks. Then may I know what is the difference between `v_src `and `v_tgt` ? I tried to use `add_property_map` and plot `v_src `and `v_tgt` separately, for `v_src ` it is obvious: they are the intersected points of tm and splitter, but for `v_tgt` I could not find them out in the output mesh.
